pure_pursuit_controller:
  ros__parameters:
    lookahead_distance: 0.7
    max_linear_speed: 0.5
    max_angular_speed: 1.0
